.github/workflows/compare-regdata.yml

```diff
@@ -76,9 +76,10 @@ jobs:
         uses: ./.github/actions/setup_lfs
 
       - name: Setup environment
-        uses: ./.github/actions/setup_env
+        uses: tardis-sn/tardis-actions/setup-env@main
         with:
           os-label: ${{ matrix.label }}
```

Comment:
is that sensible instead of saying matrix.label can we be more specific> matrix_os.label or something that makes it clear where this coming from?